### ISSUE TITLE
installation removes existing MTA

### ISSUE THREAD

**Issue by AndreaBorgia-Abo** [jprochazka/adsb-receiver#388]
I already have a MTA, namely Exim4, on my Raspi; the installation for 2.6.1 removed it and installed Postfix instead (source: bash/portal/install.sh:CheckPackage postfix ; introduced in 2.0.2)
Now I have to reinstall Exim4 :(

**Comment by jprochazka** [comment]
A while back I switched the MTA to Postfix due to the need to do additional setup on the PHP side of things in order to send mail via Exim. Postfix installs on Raspbian and Debian with no need to change things hence the reason I decided to switch out MTAs in favor of Postfix. That and it is my go to MTA when setting up servers.

To switch back to Exim at least on the package side is trivial task you can accomplish by running the following command:

`sudo apt install exim4`

However you may need to change at minimum the `sendmail_path` in your php.ini in order to make features such as password retrieval to work properly.

If/when this has changed and the PHP and the Exim package work properly together without the need to change settings I will be more than happy to simply use the default MTA which is Exim instead of Postfix however Issue #384 brought back the fact Exim does not work out of the box.

**Comment by AndreaBorgia-Abo** [comment]
That's how I switched back, I was more annoyed by the lack of prompts than anything else: if at all possible, the system should not present the owner of the system with the done deal.
I'm not really using mail from the ADSB portal, single user and all that, so I can't really comment on the other issue you mentioned. I only get logs from the Raspi via emai, not much else.

TLDR please consider making the installation of Postfix optional, if supporting Exim4 is overkill.

**Comment by jprochazka** [comment]
What I did for more advanced users such as yourself who require Exim for things other than just the ADS-B portal an option to use Exim instead of Postfix is now available by supplying a switch when executing install.sh. Both `./install.sh -m exim` and `./install.sh --mta=exim` will now skip the installation of Postfix. However, Postfix will remain the default MTA for this project at least as long as the portal remains PHP or PHP works out of the box with Exim.

_**Please keep in mind using Exim is not supported at this time and as reported in issue #384 will break things.**_

commit 90261e270049c553c4cbfb97fb6eb6bd7cb5970f
